### PR TITLE
Upgrade dependencies and code for liq 1.2

### PR DESCRIPTION
**Pull request by gbalabasquer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,7 @@ Below is the expected structure of such a config file:
   "vow_dump": "<Flop initial lot size in MKR unit>",
   "vow_bump": "<Flap fixed lot size in DAI unit>",
   "vow_hump": "<Flap Surplus buffer in DAI unit>",
+  "cat_box": "<Max total amount on auctions in DAI unit>",
```
Comment (kmbarry1) [inline README.md:74]: I would be more precise:

`<Max total DAI needed to cover all debt plus penalty fees on active auctions in DAI units>`

Comment (gbalabasquer) [reply README.md]: Solved.

File: README.md [hunk 2]
```diff
@@ -71,6 +71,7 @@ Below is the expected structure of such a config file:
   "vow_dump": "<Flop initial lot size in MKR unit>",
   "vow_bump": "<Flap fixed lot size in DAI unit>",
   "vow_hump": "<Flap Surplus buffer in DAI unit>",
+  "cat_box": "<Max total DAI needed to cover all debt plus penalty fees on active auctions in DAI units",
```
Comment (kmbarry1) [inline README.md:74]:
```suggestion
  "cat_box": "<Max total DAI needed to cover all debt plus penalty fees on active auctions in DAI unit>",
```

Comment (gbalabasquer) [reply README.md]: Done!

**Comment by gbalabasquer** [comment]
Values will be definitely adjusted when they are defined.

**Pull request merged**